v1/client_test.go

```diff
@@ -10,8 +11,24 @@ import (
 func TestDefaultClient_url(t *testing.T) {
 	client := DefaultClient()
 
-	if client.URL.String() != atlasEndpoint {
-		t.Fatalf("expected %q to be %q", client.URL.String(), atlasEndpoint)
+	if client.URL.String() != atlasDefaultEndpoint {
+		t.Fatalf("expected %q to be %q", client.URL.String(), atlasDefaultEndpoint)
+	}
+}
+
+func TestDefaultClient_urlFromEnvVar(t *testing.T) {
+	otherEndpoint := "http://127.0.0.1:1234"
+
+	err := os.Setenv("ATLAS_ADDRESS", otherEndpoint)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Unsetenv("ATLAS_ADDRESS")
```

Comment:
For maximum cleanness what you should do is `defer os.Setenv("ATLAS_ADDRESS", os.Getenv("ATLAS_ADDRESS"))` above all this.